Softeq.NetKit.Services.PushNotifications/Softeq.NetKit.Services.PushNotifications.csproj

```diff
@@ -2,15 +2,16 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>1.0.2-alpha2</Version>
+    <Version>1.1.0</Version>
     <Description></Description>
     <Authors>Softeq Development Corp.</Authors>
     <Company>Softeq Development Corp.</Company>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
-    <PackageId>Softeq.PushNotificationService</PackageId>
-    <Product>Softeq.PushNotificationService</Product>
+    <PackageId>Softeq.NetKit.PushNotifications</PackageId>
```

Comment:
I think we shouldn't change packageId and product, just increment version

Reply:
reverted